rush-plugins/rush-buildxl-graph-plugin/src/examples/debug-graph.json

```diff
@@ -0,0 +1,2379 @@
+            "name": "@rushstack/terminal (build)",
+            "commandToRun": "heft run --only build -- --clean --production --drop-graph ./src/examples/graph.json"
+          },
+          "associatedPhase": {
+            "name": "_phase:build",
+            "dependencies": {}
+          },
+          "associatedProject": {
+            "packageName": "@rushstack/terminal",
+            "projectFolder": "/repo/libraries/terminal"
+          }
+        },
+        {
+          "runner": {
+            "name": "@microsoft/rush-lib (build)",
+            "commandToRun": "heft run --only build -- --clean --production --drop-graph ./src/examples/graph.json"
+          },
+          "associatedPhase": {
+            "name": "_phase:build",
+            "dependencies": {}
+          },
+          "associatedProject": {
+            "packageName": "@microsoft/rush-lib",
+            "projectFolder": "/repo/libraries/rush-lib"
+          }
+        },
+        {
+          "runner": {
+            "name": "@rushstack/heft (build)",
+            "commandToRun": "heft run --only build -- --clean --production --drop-graph ./src/examples/graph.json"
+          },
+          "associatedPhase": {
+            "name": "_phase:build",
+            "dependencies": {}
+          },
+          "associatedProject": {
+            "packageName": "@rushstack/heft",
+            "projectFolder": "/repo/apps/heft"
+          }
+        },
+        {
+          "runner": {
+            "isNoOp": true,
+            "name": "local-node-rig (build)"
+          },
+          "associatedPhase": {
+            "name": "_phase:build",
+            "dependencies": {}
+          },
+          "associatedProject": {
+            "packageName": "local-node-rig",
+            "projectFolder": "/repo/rigs/local-node-rig"
+          }
+        }
+      ],
+      "runner": {
+        "name": "@rushstack/rush-buildxl-graph-plugin (build)",
+        "commandToRun": "heft run --only build -- --clean --production --drop-graph ./src/examples/graph.json"
+      },
+      "associatedPhase": {
+        "name": "_phase:build",
+        "dependencies": {}
+      },
+      "associatedProject": {
+        "packageName": "@rushstack/rush-buildxl-graph-plugin",
+        "projectFolder": "/repo/rush-plugins/rush-buildxl-graph-plugin"
+      }
+    },
+    {
+      "dependencies": [
+        {
+          "runner": {
+            "isNoOp": true,
+            "name": "local-eslint-config (build)"
+          },
+          "associatedPhase": {
+            "name": "_phase:build",
+            "dependencies": {}
+          },
+          "associatedProject": {
+            "packageName": "local-eslint-config",
+            "projectFolder": "/repo/eslint/local-eslint-config"
+          }
+        }
+      ],
+      "runner": {
+        "name": "@rushstack/node-core-library (build)",
+        "commandToRun": "heft run --only build -- --clean --production --drop-graph ./src/examples/graph.json"
```

Comment:
--drop-graph shouldn't have been forwarded. Did you mistakenly associate it with the phases and not just the command?